app/models/user.rb

```diff
@@ -151,14 +151,34 @@ def ensure_websites
     end
   end
 
+  # Checks if the user is at least a moderator, meaning the user is either:
+  # - a global moderator or a moderator on the current community
+  # - a global admin or an admin on the current community
+  # - has an explicit moderator privilege on the current community
+  # @return [Boolean] check result
   def is_moderator
     is_global_moderator || community_user&.is_moderator || is_admin || community_user&.privilege?('mod') || false
   end
 
+  # Checks if the user is either a global admin or an admin on the current community
+  # @return [Boolean] check result
   def is_admin
     is_global_admin || community_user&.is_admin || false
   end
 
+  # Checks if the user is neither a moderator not an admin (global or on the current community)
+  # @return [Boolean] check result
+  def is_standard
+    !is_moderator && !is_admin
+  end
+
+  # Checks if the user is either a moderator or an admin (global or on the current community).
+  # It's the inverse of the is_standard check
+  # @return [Boolean] check result
+  def is_privileged
```

Comment:
Given that we use "privilege" a lot elsewhere in the code to talk about abilities, I wonder if it would be better to rename this something like `is_superuser`

Reply:
Not applicable anymore due to extensive refactoring of access control methods